rel-eng/releasers.conf

```diff
@@ -18,3 +18,9 @@ releaser = tito.release.KojiReleaser
 autobuild_tags = foreman-plugins-nightly-nonscl-rhel7 foreman-plugins-nightly-rhel7
 builder = tito.builder.FetchBuilder
 builder.rpmbuild_options = --define "foremandist .fm1_17"
+
+[copr-nightly]
+releaser = tito.release.CoprReleaser
+project_name = @theforeman/foreman-nightly
```

Comment:
Is a project the equivalent of a repository? Or can a single project have multiple repositories?